docs/docs.json

```diff
@@ -15,34 +15,34 @@
   "navigation": {
     "versions": [
       {
-        "version": "0.2.0",
+        "version": "0.3.0",
         "groups": [
           {
             "group": "Get Started",
             "pages": [
               "common/get-started/introduction",
               "common/get-started/why",
-              "v0.2.0/get-started/usage",
+              "v0.3.0/get-started/usage",
               "common/get-started/demo"
             ]
           },
           {
             "group": "API Reference",
             "pages": [
-              "v0.2.0/api-reference/importer-props",
-              "v0.2.0/api-reference/sheet-definition-props",
-              "v0.2.0/api-reference/column-types",
-              "v0.2.0/api-reference/common-column-props",
-              "v0.2.0/api-reference/importer-state",
-              "v0.2.0/api-reference/sheet-state",
-              "v0.2.0/api-reference/validators",
-              "v0.2.0/api-reference/transformers",
-              "v0.2.0/api-reference/importer-validation-error"
+              "v0.3.0/api-reference/importer-props",
```

Comment:
Since thats a major release, you should create a new version entry and write a migration guide

Reply:
I saw you created a new files for v0.3.0. Thats good, but you also need to add a new entry in this file - not just replace 0.2.0 with 0.3.0. Look at how we have here 0.1.4 and 0.2.0. So we should maintain those and just add a new entry